### ISSUE TITLE
Translation files with redux embedded inside themes

### ISSUE THREAD

**Issue by mekshq** [reduxframework/redux-framework#1546]
Is that possible that we have all the custom text translated into our own theme .po and .mo files instead of using redux translation files and default domain?

I don't know if this is a bug you are already aware of, but for example when I use our theme text domain for all the options inside redux framework with theme predefined text domain i.e. 'my_theme' translations won't work.

For example, all the strings are generated in .po file properly but it won't read translated strings registered inside redux "sections". All the strings we use on front-end are fine, but those inside framework are not recognized somehow.

Basically, what we are trying to achieve is that our user have only one .po and .mo translation file kept inside our theme "languages" folder with our textdomain which will keep both frontend and backend text. To make this more clear, I'm not talking about your predefined translations for save button, reset buttons, media uploads etc(we still want to keep them inside your redux languages folder)... I'm talking about our custom text we enter while we generate all the options inside framework. 

Cheers

**Comment by Joseph-a** [comment]
Hi,
I added "load_theme_textdomain" before the redux without any hook, but in Russian language we will have this error! What do you think? It happened just for Russian and the other languages work like a charm.
Fatal error: Maximum execution time of 30 second exceeded in  MY_PATH\inc\class.redux_api.php line 272

We increased the limitation of php execution time but nothing happened.

**Comment by dovy** [comment]
Seems like a memory limit on your host and you're doing too much.  :P

**Comment by Joseph-a** [comment]
But why this happened when I changed the language to Russian!?? In the other languages it works without any errors.

**Comment by dovy** [comment]
I honestly have no idea. That sounds like a WP core/memory issue. Nothing in Redux should affect that at all. It's just a language pack.  :(
